### ISSUE TITLE
Firefox plugin is no longer available on Firefox website

### ISSUE THREAD

**Issue by avagraphique** [keeweb/keeweb-connect#51]
Link from app / github to the Firefox keeweb-connect plugin page shows page not found.
 
https://addons.mozilla.org/en-US/firefox/addon/keeweb-connect/

**Comment by ticpu** [comment]
That's quite a blocker when reinstalling an OS :/

Is there a documentation for manual install?

**Comment by HarlemSquirrel** [comment]
@Aetherinox do you think this is because the last version was not updated for so long that Firefox removed it?

**Comment by antelle** [comment]
I remember I received an email that it would be auto-deleted. Also it start getting bad reviews because of not being supported, so I assume it needs a re-test, an update, and a publish

**Comment by HarlemSquirrel** [comment]
I was able to run the extension after building locally on the master branch with `npm start` and trying it out in `about:debugging` with Firefox 124.0.1 on Pop!_OS 22.04 by pointing to `dist/firefox/manifest.json`

**Comment by antelle** [comment]
Btw I just received an e-mail about the Chrome extension:

> Beginning June of this year, we will begin to [gradually disable extensions running Manifest V2](https://notifications.google.com/g/p/ANiao5p7MjocY-nb8xaOJ0RUPq7ZJ7ZRsBKThrRQijIc5M7qgdiEHEJStKEJPwMc4tA6a2DWaBE1VctjCePt0_PuBpHspvJtc2joSMPzEd37JS6m9Ztol31l4G82dwG1b546P0GyloEBDe-etqEce72e9ZXbUlknt4v9kdKliOC3kbzYdyMk42t7O13f8nXDSC15PtDrOb9na4FeJMMMLlwFHPMR88JYCxw_mH9GnvSzrp_fJv5z_5bg15G4yqo9dok8EhQlR3JnIsfKrGBAgquS6yp83Q) for Chrome users.

**Comment by Aetherinox** [comment]
Yeah, at the very least, the plugin would have been removed due to Manifest v2 -> v3. Google has been trying to push out v3 for years now.  

So I'm sure it needs an update to conform with the newest standards.
